doc/release/release_notes_v1_2.md

```diff
@@ -0,0 +1,328 @@
+ - Deleting the container (if the pool does not grant the user permission).
+ - Getting and setting container properties.
+ - Getting and modifying the container ACL.
+ - Modifying the container's owner.
+
+This is reflected in the set of supported [container permissions](https://daos-stack.github.io/overview/security/#permissions).
+
+[More information](https://daos-stack.github.io/user/container/#access-control-lists)
+
+
+
+### Improved Control Plane
+
+#### External Device States (via DMG)
+
+The device states that are returned from a device query by the administrator are dependent on both the persistently stored device state in SMD, and the in-memory BIO device list.
+
+ - NORMAL: A fully functional device in use by DAOS (or in setup).
+ - EVICTED: A device has been manually evicted and is no longer in use by DAOS.
+ - UNPLUGGED: A device previously used by DAOS is unplugged.
+ - NEW: A new device is available for use by DAOS.
+
+[More information](https://daos-stack.github.io/admin/administration/#nvme-ssd-health-monitoring)
+
+#### User Interfaces
+
+The DAOS dmg management utility can be used to query NVMe SSD health data, as well as provide useful commands for NVMe SSD eviction, hot plug, and device identification. NVMe hot plug and reintegration will not be fully supported until DAOS 2.2 release. 
+
+ - Manually evict and NVMe SSD
+ - Replace an evicted SSD with a new NVMe SSD
+ - Reuse a faulty NVMe SSD (w/o reintegration)
+ - Locate a healthy NVMe SSD (for VMD-enabled devices only)
+ - Locate an evicted NVMe SSD (for VMD-enabled devices only)
+
+[More information](https://daos-stack.github.io/admin/administration/#nvme-ssd-eviction-and-hotplug)
+
+
+
+
+### Replication and Self Healing
+
+In DAOS, if the data is replicated with multiple copies on different targets, once one of the targets fail, the data on it will be rebuilt on the other targets. This reduces the data redundancy that would be impacted by the target failure. 
+
+[More information]( https://daos-stack.github.io/admin/pool_operations/#target-exclusion-and-self-healing)
+
+### Erasure Code (Preview)
+
+This release implements Reed Solomon based EC for data protection of DAOS, it supports EC data recovery for storage target failure, and data migration while extending the storage pool. Main features of DAOS EC include:
+
+ - Reed-Solomon based EC support for I/O. DAOS has two value types: array value for large streaming I/O; single value for Key-value store. DAOS EC supports both array value and single value. An array value always has fixed stripe size and cell size, whereas a single value has dynamic stripe size to provide storage efficiency of application data with arbitrary size. This feature also provides a set of predefined EC object classes as its user interface. The user can choose the appropriate object class for their data based on their requirements and restrictions. For example, capacity and bandwidth overhead, fault tolerance level, etc.
```

Comment:
This feature also provides a set of predefined EC object classes as its user interface.  Perhaps instead:

The EC feature is activated by utilizing one of the predefined EC object classes.